test/new_project.sh

```diff
@@ -16,7 +20,9 @@ rm  -rf tmp-project
 # test generating new project in new dir
 ../bin/near new_project 'tmp-project'
 cd tmp-project
-npm install
-npm uninstall near-shell
-npm install ../../
-NODE_ENV=development npm run test
\ No newline at end of file
+FILE=package.json
+if test -f "$FILE"; then
+  echo "${GREEN}$FILE exists. Have a cookie!"
+else
+  echo "${RED}ERROR: file not found."
```

Comment:
need to add `exit 1` or it won't return error code (and so won't fail CI)

Reply:
done